packages/aws-cdk-lib/scripts/airlift-custom-resource-handlers.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+scriptdir=$(cd $(dirname $0) && pwd)
+customresourcedir=${scriptdir}/../../custom-resource-handlers
```

Comment:
I'd rather you not make assumptions on where the SOURCE directory for this lives, but instead use Node's module resolution mechanism to find the path of this library:

```
customresourcedir=$(node -p "path.dirname(require.resolve('@aws-cdk/custom-resource-handlers/package.json'))")
```